### PR TITLE
geom_jitter adjustments

### PR DESCRIPTION
**Pull request by Lunalo**
@rdstern 

This not fully fixed but now Violin can be seen

### CODE REVIEW

File: instat/static/InstatObject/R/data_object_R6.R [hunk 1]
```diff
@@ -84,7 +84,7 @@ data_object <- R6Class("data_object",
                             variables_metadata_changed = function(new_value) {
                               if(missing(new_value)) return(private$.variables_metadata_changed)
                               else {
-                                if(new_value != TRUE && new_value != FALSE) stop("new_val must be TRUE or FALSE")
+                                if(new_value != TRUE && new_geom_jittervalue != FALSE) stop("new_val must be TRUE or FALSE")
```
Comment (dannyparsons) [inline instat/static/InstatObject/R/data_object_R6.R]: What is this change?

**Comment by Lunalo** [comment]
I will fix this

**Comment by lilyclements** [comment]
@Lunalo How is this going?

**Comment by stevekogo** [comment]
@Lunalo How is this going?

**Comment by Lunalo** [comment]
This is ready

**Comment by lilyclements** [comment]
Can you give a brief explanation of what this fixes so I know what to look for? I don't understand the link between geom_jitter and start of rains

**Comment by maxwellfundi** [comment]
@dannyparsons  you can now accept this

**Comment by lilyclements** [comment]
Was the crash problem fixed?

**Comment by dannyparsons** [comment]
Yes it should be, but could be checked.

**Pull request merged**